src/components/speakerDetails.re

```diff
@@ -0,0 +1,25 @@
+open Util;
+
+open Data;
+
+[@bs.module] external style : Js.t({..}) = "./speakerDetails.module.scss";
+
+let component = ReasonReact.statelessComponent("SpeakerDetails");
+
+let make = (~speaker: Data.speakerData, _children) => {
+  ...component,
+  render: _self =>
+    <div className=style##root key=speaker.name>
```

Comment:
Looks like we could reuse speakerCard component here and extend it to have a prop for showing full/card view since the HTML is almost the same?

Reply:
I used the SpeakerCard component before... it doesn't make sense to put the name on the bottom of the image, since the name should be prominent with a <h1> tag on top